src/kubectl-helm-minikube/install.sh

```diff
@@ -167,6 +174,17 @@ if [ ${HELM_VERSION} != "none" ]; then
     tmp_helm_filename="/tmp/helm/${helm_filename}"
     curl -sSL "https://get.helm.sh/${helm_filename}" -o "${tmp_helm_filename}"
     curl -sSL "https://github.com/helm/helm/releases/download/${HELM_VERSION}/${helm_filename}.asc" -o "${tmp_helm_filename}.asc"
+    if grep -q "BlobNotFound" "${tmp_helm_filename}"; then
+        echo -e "\n(!) Failed to fetch the latest artifacts for helm ${HELM_VERSION}..."
+        repo_url=https://api.github.com/repos/helm/helm/releases
+        requested_version=$(get_previous_version "${repo_url}")
+        echo -e "\nAttempting to install ${requested_version}"
+        helm_filename="helm-${requested_version}-linux-${architecture}.tar.gz"
+        tmp_helm_filename="/tmp/helm/${helm_filename}"
+        curl -sSL "https://get.helm.sh/${helm_filename}" -o "${tmp_helm_filename}"
+        curl -sSL "https://github.com/helm/helm/releases/download/${requested_version}/${helm_filename}.asc" -o "${tmp_helm_filename}.asc"
```

Comment:
This command looks repetitive, how about moving it out of the condition block? and syncing this line with :176 (`curl -sSL "https://github.com/helm/helm/releases/download/${HELM_VERSION}/${helm_filename}.asc" -o "${tmp_helm_filename}.asc"` 

Both commands are similar, we only need to update variables

Reply:
have updated ! thanks 👍